umbraco-cloud/set-up/project-settings/manage-hostnames/rewrites-on-cloud.md

```diff
@@ -25,24 +25,23 @@ When you are doing rewrite rules on Umbraco Cloud there are a few important thin
 * Always make sure that you add a condition that negates the Umbraco Backoffice - `/umbraco`, otherwise, you will not be able to do deployments to/from the environment
 * To be able to continue working locally with your Umbraco Cloud project, you also need to add a condition that negates `localhost`
 
-## Hiding the Default umbraco.io Url
+## Hiding the default umbraco.io URL
 
-Once you've assigned a hostname to your Live environment, you may want to "hide" the project's default URL (e.g. mysite.s1.umbraco.io) for different reasons. Perhaps for SEO or to make it clear to your users that the site can be accessed using only one hostname.
+Once you've assigned a hostname to your Live environment, you may want to "hide" the project's default URL (e.g. example.euwest01.umbraco.io) for different reasons. Perhaps for SEO or to make it clear to your users that the site can be accessed using only the primary hostname.
 
-One approach for this is to add a new rewrite rule to the `<system.webServer><rewrite><rules>` section in the `web.config` file. For example, the following rule will redirect all requests for the project mysite.euwest01.umbraco.io URL to the mysite.com URL and respond with a permanent redirect status.
+One approach for this is to add a new rewrite rule to the `<system.webServer><rewrite><rules>` section in the `web.config` file. For example, the following rule will redirect all requests for the project example.euwest01.umbraco.io URL to the example.com URL (using HTTPS and including the `www.` prefix) and respond with a permanent redirect status.
 
 ```xml
-<rule name="Redirects umbraco.io to actual domain" stopProcessing="true">
+<rule name="Redirect umbraco.io to primary hostname" stopProcessing="true">
   <match url=".*" />
   <conditions>
-    <add input="{HTTP_HOST}" pattern="^(.*)?.euwest01.umbraco.io$" />
-    <add input="{REQUEST_URI}" negate="true" pattern="^/umbraco" />
-    <add input="{REQUEST_URI}" negate="true" pattern="^/DependencyHandler.axd" />
-    <add input="{REQUEST_URI}" negate="true" pattern="^/App_Plugins" />
-    <add input="{REQUEST_URI}" negate="true" pattern="localhost" />
+    <add input="{HTTP_HOST}" pattern="\.umbraco\.io$" />
```

Comment:
Based on a previous PR, that also introduced some new changes to this article, specifically the following rule:
`<add input="{HTTP_HOST}" pattern="^(dev|stage)(.*)?.euwest01.umbraco.io$" ignoreCase="true" negate="true" />` with pattern matching, however it is still using the euwest01 tag. 

I am wondering whether changing it to <add input="{HTTP_HOST}" pattern="^(dev|stage)(.*)?\.umbraco\.io$" would be possible so that it can be used in all regions without making any changes? 🤔

Reply:
I've resolved the merge conflict caused by this change, so any domain starting with either `dev.` or `stage.` and ending with `.umbraco.io` will be ignored by this rewrite rule 👍🏻

I do want to note that development and staging environments can also have their own custom hostnames, so users may want to create environment specific rewrite rules if they want to add redirects for those environments as well. This rule does make sense to have by default though, because you won't get redirected to the primary hostname of the live environment after creating a new development/staging environment 👌🏻